nuxt.config.ts

```diff
@@ -1,54 +1,34 @@
 import { defineNuxtConfig } from 'nuxt/config'
 
-import Joi from 'joi'
 import pkg from './package.json'
 
-const config = {
-  apiUrl: process.env.API_BASE_URL,
-  mapboxAccessToken: process.env.MAPBOX_ACCESS_TOKEN,
-  mapboxStyle: process.env.MAPBOX_STYLE,
-  fontAwesomeProkitKey: process.env.FONT_AWESOME_PRO_KIT_KEY,
-  initGoogleServices: process.env.INIT_GOOGLE_SERVICES === 'true'
-}
-
-const configSchema = Joi.object({
-  apiUrl: Joi.string().uri().required(),
-  mapboxAccessToken: Joi.string().required(),
-  mapboxStyle: Joi.string().required(),
-  fontAwesomeProkitKey: Joi.string().required(),
-  initGoogleServices: Joi.boolean().required()
-}).strict()
-
-// Validate configuration
-const { error } = configSchema.validate(config)
-if (error) {
-  throw new Error(`Configuration error: ${error.message}`)
-}
-
 export default defineNuxtConfig({
   // devtools: { enabled: true },
   spaLoadingTemplate: false,
   ssr: false,
   build: {
-    transpile: ['vuetify']
+    transpile: ['vuetify'],
   },
-  css: ['vuetify/lib/styles/main.sass', '@mdi/font/css/materialdesignicons.min.css'],
+  css: [
+    'vuetify/lib/styles/main.sass',
+    '@mdi/font/css/materialdesignicons.min.css',
+  ],
   devServer: {
-    port: 3001
+    port: 3001,
   },
   modules: ['@pinia/nuxt', 'nuxt-snackbar', '@vueuse/nuxt', '@nuxt/eslint'],
   snackbar: {
     bottom: true,
     duration: 2000,
     dense: true,
-    zIndex: 99999
+    zIndex: 99999,
   },
   runtimeConfig: {
     public: {
-      apiUrl: config.apiUrl,
-      mapboxAccessToken: config.mapboxAccessToken,
-      mapboxStyle: config.mapboxStyle,
-      initGoogleServices: config.initGoogleServices,
+      apiBaseUrl: process.env.NUXT_PUBLIC_API_BASE_URL,
```

Comment:
(update .env.template accordingly)

Reply:
(template updated, thanks for pointing it out)